gui-libs/wf-config/wf-config-9999.ebuild

```diff
@@ -0,0 +1,46 @@
+# Copyright 2019-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit meson
+
+DESCRIPTION="library for managing wayfire configuration files"
+HOMEPAGE="https://github.com/WayfireWM/wf-config"
+
+if [[ ${PV} == 9999 ]]; then
+	inherit git-r3
+	EGIT_REPO_URI="https://github.com/WayfireWM/${PN}.git"
+else
+	SRC_URI="https://github.com/WayfireWM/${PN}/releases/download/${PV}/${P}.tar.xz"
+	KEYWORDS="~amd64 ~arm64 ~x86"
```

Comment:
You should probably have KEYWORDS="" in the 9999 branch and drop this one.

Reply:
I was actually told to do the other way round, remove the empty and keep the one with the names: https://github.com/gentoo/gentoo/pull/15483#discussion_r415301039

Reply:
I think me and juippis agree? This is for the 9999 ebuild. The tool messes up when 2 exist in one ebuild.

Reply:
I am unsure.... maybe we are interpreting it differently.  
What I am thinking is that if the 9999 version is the one installed then keywords is empty by default so it works and if 0.4.0 is installed then it sets it to the keywords (as defined).

Is this not what he meant?

Also I am kind of confused when you say only one. Like are you saying it should only be defined once? (that is the case here, though through the program flow it could be 0 depending on ${PV}.

I am getting a tad confused.

If we can get my misunderstanding cleared, it would be nice :confused:

Reply:
RIght, okay, I get you. It defaults to blank so it's okay. I was saying shift it to the other one and call it blank but it doesn't really matter here.

Sorry for the mess!

Reply:
awesome! so i hope this looks fine :smile_cat: